Add attachments limitation (#32)
elasticio/csv-component: README.md

The output of the CSV Write component will be a message with an attachment.  In
order to access this attachment, the component following the CSV Write must be
able to handle file attachments.

### Limitations

1. You may get `Component run out of memory and terminated.` error during run-time, that means that component needs more memory, please add

>  `EIO_REQUIRED_RAM_MB` Environment variables for csv-component in this case.

Reviewer: ```suggestion
 `EIO_REQUIRED_RAM_MB` environment variable with an appropriate value (e.g. value `512` means that 512 MB will be allocated) for the component in this case.
```

> 2. Maximal possible size for attachment is 10 Mb.

Reviewer: ```suggestion
2. Maximal possible size for an attachment is 10 MB.
```

> 3. Attachments mechanism does NOT work with  [Local Agent Installation](https://support.elastic.io/support/solutions/articles/14000076461-announcing-the-local-agent-)

Reviewer: ```suggestion
3. Attachments mechanism does not work with [Local Agent Installation](https://support.elastic.io/support/solutions/articles/14000076461-announcing-the-local-agent-)
```
